pkg/virt-controller/watch/migration.go

```diff
@@ -1038,6 +1105,13 @@ func (c *MigrationController) sync(key string, migration *virtv1.VirtualMachineI
 		return fmt.Errorf("vmi is inelgible for migration because another migration job is running")
 	}
 
+	if migration.DeletionTimestamp != nil {
+		err = c.markMigrationAbortInVmiStatus(migration, vmi)
+		if err != nil {
+			return err
+		}
```

Comment:
don't update the Status in the sync() function.

Reply:
This function (in its original name, `handleSignalMigrationAbort()`, was called from sync() before my changes.
I moved it back to its original place under `case Running` though.